app/assets/stylesheets/cortex-plugins-core/components/thumbnail-placeholder.scss

```diff
@@ -0,0 +1,12 @@
+.thumbnail-placeholder {
+  background-color: #E4E4E4; // TODO: Abstract to cortex-style-base library. This should be $color-grey-evenlighter
```

Comment:
Sure - I'll just copy all variables from `cortex` to `cortex-plugins-core`, with the expectation that these need to be removed once `cortex-plugins-core` can pull from a `cortex-style-base`. Thanks!